crates/networking/p2p/src/bootnodes.rs

```diff
@@ -19,12 +23,25 @@ impl FromStr for Bootnodes {
         match s {
             "default" => Ok(Bootnodes::Default),
             "none" => Ok(Bootnodes::None),
-            _ => s
-                .split(',')
-                .map(Enr::from_str)
-                .collect::<Result<Vec<_>, String>>()
-                .map(Bootnodes::Custom)
-                .map_err(|err| anyhow!("Failed to parse bootnodes: {err:?}")),
+            _ => {
+                if let Ok(enrs) = s
+                    .split(',')
+                    .map(Enr::from_str)
+                    .collect::<Result<Vec<_>, _>>()
+                {
+                    return Ok(Bootnodes::Custom(enrs));
+                }
+
+                if let Ok(addrs) = s
+                    .split(',')
+                    .map(Multiaddr::from_str)
+                    .collect::<Result<Vec<_>, _>>()
+                {
+                    return Ok(Bootnodes::Multiaddr(addrs));
+                }
+
+                Err(anyhow!("Failed to parse bootnodes as ENR or Multiaddr"))
+            }
```

Comment:
we should probably add a prefix like enr:xyz and multi:abc, so that the error here isn't ambiguous